webapp-django/crashstats/crashstats/jinja2/crashstats/crashes_per_day.html

```diff
@@ -43,7 +43,7 @@
                         <td>
                             <label class="visually-hidden" for="daily_search_version_form_products">Select Product</label>
                             <select id="daily_search_version_form_products" name="p">
-                            {% for product_name in releases %}
+                            {% for product_name in products %}
```

Comment:
We should avoid the word "releases" and "releases" as much as possible. We should just talk about "product versions" or "products".